apps/examples/websocket/websocket_main.c

```diff
@@ -795,10 +795,6 @@ int websocket_main(int argc, char *argv[])
 		goto error_with_function_failure;
 	}
 
-	if (input) {
-		free(input);
-	}
-
 	return 0;
```

Comment:
what if it never goes to the `error_with_function_failure:` label.?

Reply:
As mentioned above, if the functions above are executed, "input" will be freed. So in the case of success, it is not necessary to free input. Please let me know if the reasoning is correct.